app/services/manifest_builder.rb

```diff
@@ -460,10 +460,10 @@ def geo_image?(member)
   end
 
   class Numismatics::IssueNode < CollectionNode
-    # Only include coins which have file sets;
+    # Only include Coin resources which have file sets;
     # otherwise there is no image for the viewer.
     def members
-      @members ||= super.select { |coin| coin.member_ids.present? }
+      @members ||= super.select { |r| r.is_a? Numismatics::Coin }.select { |coin| coin.member_ids.present? }
```

Comment:
I think this would be clearer if we pulled them directly from a decorator method and reimplemented the bit of permissions code that's being reused by `super`.

Reply:
Pushed logic to the Issue decorator.